src/go/k8s/controllers/redpanda/cluster_controller_configuration.go

```diff
@@ -372,7 +372,7 @@ func mapStatusToCondition(
 				Type:    redpandav1alpha1.ClusterConfiguredConditionType,
 				Status:  corev1.ConditionFalse,
 				Reason:  redpandav1alpha1.ClusterConfiguredReasonUpdating,
-				Message: fmt.Sprintf("Node %d needs restart", nodeStatus.NodeID),
+				Message: fmt.Sprintf("Node %d needs restart", nodeStatus.NodeID), // actually pod ordinal
```

Comment:
I still see the comment. Should it be removed as https://github.com/redpanda-data/redpanda/pull/7581#discussion_r1049962886